src/test/ruby/ssl/fixtures/othername_cert/othername.cnf

```diff
@@ -0,0 +1,15 @@
+[ req ]
```

Comment:
I couldn't work out how to generate a test certificate with the `otherName` extension programatically, so used `openssl` on the command line with this config file.

```
openssl req -newkey rsa:2048 -nodes -x509 -days 36500 -keyout othername.key -out othername.crt -config othername.cnf -extensions req_ext
```